Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon_test.py

```diff
@@ -2823,3 +2832,136 @@ def test_get_endpint_command(requests_mock, mocker):
     context = result.get('EntryContext')
 
     assert context['Endpoint(val.ID && val.ID == obj.ID)'] == [endpoint_context]
+
+
+def test_create_hostgroup_invalid(requests_mock):
+    """
+    Test Create hostgroup with valid args with unsuccessful args
+    Given
+     - Invalid arguments for hostgroup
+    When
+     - Calling create hostgroup command
+    Then
+     - Throw an error
+     """
+    from CrowdStrikeFalcon import create_host_group_command
+    response_data = load_json('test_data/test_create_hostgroup_invalid_data.json')
+    requests_mock.post(
+        f'{SERVER_URL}/devices/entities/host-groups/v1',
+        json=response_data,
+        status_code=400,
+        reason='Bad Request'
+    )
+    with pytest.raises(SystemExit):
+        create_host_group_command(name="dem test",
+                                  description="dem des",
+                                  group_type='static',
+                                  assignment_rule="device_id:[''],hostname:['falcon-crowdstrike-sensor-centos7']")
+
+
+def test_update_hostgroup_invalid(requests_mock):
+    """
+    Test Create hostgroup with valid args with unsuccessful args
+    Given
+     - Invalid arguments for hostgroup
+    When
+     - Calling create hostgroup command
+    Then
+     - Throw an error
```

Comment:
You're catching the `SystemExit` - so that means that actually `return_error` was called specifically, and it exited the code. 

You could also mock `return_error` and assert that it was called but this test is ok as well.